feat: Support proxy config and env vars for HTTP, TCP and TLS Probers (#194)
megaease/easeprobe: README.md

    host: example.com:22
    timeout: 10s # default is 30 seconds
    interval: 2m # default is 60 seconds

>     proxy: socks5://proxy.server:1080 # Optional. Only support sock5.

Reviewer: ```suggestion
    proxy: socks5://proxy.server:1080 # Optional. Only support socks5.
```

  tls:
    - name: expired test
      host: expired.badssl.com:443

>       proxy: socks5://proxy.server:1080 # Optional. Only support sock5.

Reviewer: ```suggestion
      proxy: socks5://proxy.server:1080 # Optional. Only support sock5.
```
```suggestion
      proxy: socks5://proxy.server:1080 # Optional. Only support socks5.
```

        - "KEY=Value"
```

> **Note**:
>

> > 1) Setting the environment variables - `$HTTP_PROXY` & `$HTTPS_PROX` to configure the proxy for HTTP Webhook notification.

Reviewer 2: ```suggestion
> 1) Setting the environment variables `$HTTP_PROXY` & `$HTTPS_PROXY` allows for configuring the proxy settings for all HTTP related webhook notifications such as discord, slack, telegram etc.
```
